src/biteme/__main__.py

```diff
@@ -1,37 +1,5 @@
-from __future__ import annotations
-
-from pathlib import Path
-from typing import Optional
-
-import typer
-
-from biteme import pybites
-
-
-__all__ = ["cli"]
-
-
-cli = typer.Typer(context_settings={"auto_envvar_prefix": "PYBITES"})
-
-
-@cli.command()
-def info(bite: int) -> None:
-    bite_info = pybites._bite_info(bite)
-    typer.echo(f"{bite_info=}")
-
-
-@cli.command()
-def download(
-    bite: int = typer.Argument(...),
-    directory: Optional[Path] = typer.Argument(
-        None,
-        writable=True,
-        file_okay=False,
-    ),
-    api_key: Optional[str] = typer.Option(None),
-) -> None:
-    pybites.download_bite(bite, directory, api_key)
+import biteme
```

Comment:
I am wondering when to still use `__main__.py` when you have configured `tool.poetry.scripts` in your toml file?

Reply:
It's redundant but it can be helpful so that you can run it as `/path/to/python -m biteme` and _know_ that you're running it inside of that environment.

Reply:
Yeah I do like that use case.

Reply:
Going to put it back in for now. A lot of popular Python CLI tools have this behavior, including `pip`.